apps/opik-backend/pom.xml

```diff
@@ -54,9 +55,14 @@
 
     <dependencies>
         <dependency>
-            <groupId>com.newrelic.agent.java</groupId>
-            <artifactId>newrelic-api</artifactId>
-            <version>8.14.0</version>
+            <groupId>io.opentelemetry.instrumentation</groupId>
+            <artifactId>opentelemetry-instrumentation-annotations</artifactId>
+            <version>${opentelmetry.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.opentelemetry.instrumentation</groupId>
+            <artifactId>opentelemetry-r2dbc-1.0</artifactId>
+            <version>${opentelmetry.version}-alpha</version>
```

Comment:
I found mutiple opentelemetry libraries for R2DBC. This seems to be an early one in alpha state. Why this choice over others?

Reply:
This is the agent instrumentation, and all modules are in alpha: https://mvnrepository.com/artifact/io.opentelemetry.instrumentation/opentelemetry-r2dbc-1.0. Here is the definition of alpha:

https://github.com/open-telemetry/opentelemetry-java/blob/main/VERSIONING.md#stable-vs-alpha. Basically, they mention such modules don't have backward compatibility guarantees and a few other problems that may require changes from version to version, but that doesn't mean they are not mature.